src/search/cost_test_ebpf.cc

```diff
@@ -26,12 +28,93 @@ double get_error_cost(inst* p1, inst* p2, int win_start, int win_end) {
   return c.error_cost(&prog1, inst::max_prog_len, &prog2, inst::max_prog_len);
 }
 
+double get_safety_cost(inst* p1, inst* p2, int win_start, int win_end) {
```

Comment:
The new tests are used to test safety_cost_repair. Maybe this function can be removed?